### DISCUSSION TITLE
Maintainer team for stdlib

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/stdlib#484] [General]
This is a follow up from https://fortran-lang.discourse.group/t/1506 and https://github.com/fortran-lang/stdlib/discussions/447.

To get the ball rolling I'm starting this thread here. I'd propose the following strategy:

1. anyone interested in taking up maintainer responsibility for stdlib can volunteer here
2. we create a (public) maintainer team for the respective projects in GitHub,
    this allows tagging `@fortran-lang/stdlib` or similar in issues or PRs
3. we review the repository for:
    - updating the contribution guidelines, README and docs
    - adding a (near future) roadmap to the project wiki

The scope for the stdlib maintainers was briefly discussed at the [July 2021 monthly call](https://fortran-lang.discourse.group/t/1510/9):

- responsible to search and assign reviewers for PRs (make sure no PR is overlooked, ...)
- help new contributors with their first contribution (or find / assign somebody for this)
- help new reviewers with their first review (or find / assign somebody for this)
- keep the issue tracker and the PR labels up-to-date (or find / assign somebody for this)
- set a project road-map and prioritize parts of stdlib development

Anything else to add here?

Preferably we find a team of three or more that are interested to take a lead here and organize stdlib.

**Comment by milancurcic** [comment]
Thanks, @awvwgk, I volunteer to be on the maintainer team.

**Comment by jvdp1** [comment]
Thanks @awvwgk. I was on holidays the last few weeks, and it seems that I have a lot to catch up :)

Sounds good to me, and well needed. I volunteer to be on the maintainer team.
It would maybe good to assign some topics to each maintainer (e.g. based on their background/preferences/...) to check if all possible topics in `stdlib` are covered and to be sure that all issues will be considered  by at least one maintainer.

**Comment by Jim-215-Fisher** [comment]
I am available to be on the maintainer team if needed.

**Comment by milancurcic** [comment]
@ivan-pi you've been quite active with stdlib since the start. Can we count on you as a maintainer?

**Reply by ivan-pi** [reply]
Yes, I was planning to reply soon.

**Comment by awvwgk** [comment]
I created @fortran-lang/stdlib, this team can now be mentioned here and requested for reviews to ping all members.

**Comment by jalvesz** [comment]
Hi, I wanted to know if the team is open to taking more volunteers? If so, I would like to join :)

**Comment by jvdp1** [comment]
@jalvesz additional help will be appreciated. So I support it!

@milancurcic @14NGiestas @Jim-215-Fisher @gareth-nx Do you agree to add @jalvesz to the stdlib maintainer team?

**Reply by milancurcic** [reply]
I fully agree, great addition, and I should be removed due to inactivity. Thank you @jalvesz!!

**Reply by jvdp1** [reply]
@milancurcic Could you add @jalvesz to the maintainer team, please (I don't have the rights to do it). Thanks!

**Reply by jalvesz** [reply]
Hello everyone! Thanks for the support @jvdp1 @milancurcic 🙂 just wanted to check if there's something I should do on my side to join the team in case everyone is in favor?
